### ISSUE TITLE
OpenVINO backend

### ISSUE THREAD

**Issue by likholat** [scverse/scvi-tools#1485]
I would like to contribute an [OpenVINO](https://docs.openvino.ai/latest/index.html) backend (allows to speed up the runtime of pretrained models) for the models in this repository.

Now I have a script, which converts [VAE](https://github.com/scverse/scvi-tools/blob/master/scvi/module/_vae.py) model to OpenVINO representation. Could you please share tests or demo for VAE model so I can compare accuracy and execution time of the original and OpenVINO model?

**Comment by adamgayoso** [comment]
Demo is [here](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/harmonization.html). Would you be able to provide some details on the advantages of OpenVINO for this project? We are not typically in the scenario of serving pre-trained models. Thanks!

**Comment by adamgayoso** [comment]
Closing as this is out of scope for this package.
